SEP-0002.md

```diff
@@ -3,203 +3,288 @@
+
+* Current commit access to the core repo, or any accepted affiliated package maintained by the project, and with a commit to that repository within the last 12 months.
+* Currently holds a role listed under the community roles section of the website: <https://sunpy.org/project/##community-roles>.
+* Is a currently serving Advisory Board member.
+
+Applying this set of criteria on 2024/04/09 resulted in the following people being the initial Project Custodians:
+
+* Stuart Mumford
+* Will Barnes
+* Laura Hayes
+* David Pérez-Suárez
+* Conor MacBride
+* Monica Bobra
+* Steven Christe
+* Russell Hewett
+* David Perez-Suarez
+* Tiago M. D. Pereira
+* Kevin Reardon
+* Albert Y. Shih
+* Nabil Freij
+* David Stansby
+* Alasdair Wilson
+* Daniel Ryan
+* Shane Maloney
+
+#### Adding New Project Custodians
+
+Anyone is eligible to be nominated to be a Project Custodian, as long as they meet the criteria laid out in the Membership section above.
+Self-nominations are encouraged.
+
+The process for adding a new Project member is as follows:
+
+1. A nomination is received by the Steering Committee, a nomination should include a short, factual, statement stating how the nominee meets the criteria for Project Custodianship.
+The Steering Committee, and the Ombudsperson check that the nomination is factually accurate, their record of contribution to the community adheres to the Code of Conduct, and they wish to serve as a Project Custodian (if not self-nominated).
+1. The candidate is then put to the Project Custodians.
+The name and nomination statement of the candidate will be shared with the Project Custodians.
+The candidate can become a Project Custodian if, during the two weeks following the candidate being put to the Project Custodians, five existing Project Custodians approve the candidate and no Project Custodians call for an election.
+1. Project Custodians can call for an election either by informing the Steering Committee or the Ombudsperson.
+If a vote is called for then the candidate will become a Project Custodian with a simple majority of all votes cast in the election.
+An election needs to be announced to the Project Custodians one week before the voting opens and voting will be open for two weeks.
+1. The candidate will be informed promptly of the outcome by the Steering Committee.
+
+The name of a candidate should not be shared outside of the Project Custodians until they become a Project Custodian.
+A public list of Project Custodians will be kept on the SunPy website.
+
+Nominations, approvals and any elections of Project Custodians should take place before the initial election of the Steering Committee.
+
+#### Active and Emeritus Status
+
+Project Custodians have no term limits.
+Project Custodians who are no longer actively contributing to the project should contact the Steering Committee to move themselves to emeritus status.
```

Comment:
This section is a bit frustrating to read because "active" as a status is not clearly defined and delineated, so you end up implicitly discussing non-active (in terms of contributions) active (in terms of status) custodians and active (in terms of contributions) non-active (i.e., emeritus) custodians.